third_party/Makefile

```diff
@@ -81,10 +80,11 @@ rocm_rocblas: sb_micro_path
 ifeq (, $(wildcard $(SB_MICRO_PATH)/bin/rocblas-bench))
 	if [ -d rocBLAS ]; then rm -rf rocBLAS; fi
 	git clone -b ${ROCM_VERSION} https://github.com/ROCmSoftwarePlatform/rocBLAS.git ./rocBLAS
+# For rocm4.0.0, due to some issues about rocblas building, now only support MI100(gfx908) 
 ifeq (${ROCM_VERSION}, rocm-4.0.0)
```

Comment:
rocBLAS 4.0.0 can't be built with arches except for gfx908? How can we use or skip rocBLAS for other arches?